bridgesync/processor.go

```diff
@@ -575,6 +610,73 @@ func (p *processor) GetClaimsPaged(
 	return claims, claimsCount, nil
 }
 
+// GetUnsetClaimsPaged returns a paginated list of unset claims
+func (p *processor) GetUnsetClaimsPaged(
+	ctx context.Context, pageNumber, pageSize uint32,
+	networkIDs []uint32, globalIndex *big.Int,
```

Comment:
AFAICT, the `networkIDs` is unused, so it can be removed?
```suggestion
	globalIndex *big.Int,
```